packages/connect/src/api/firmware/x509certificate.ts

```diff
@@ -98,13 +98,61 @@ const derBitStringValue = (byteArray: Uint8Array) => ({
     bytes: byteArray.subarray(1),
 });
 
+export const parseSignature = (byteArray: Uint8Array) => {
+    const { unusedBits, bytes } = derBitStringValue(byteArray);
+    const asn1 = derToAsn1(bytes);
+
+    if (asn1.cls !== 0 || asn1.tag !== 16 || !asn1.structured) {
+        throw new Error('Bad SPKI. Not a SEQUENCE.');
```

Comment:
Copy-paste error. SPKI stands for "SubjectPublicKeyInfo", this should say "Bad signature. Not a SEQUENCE."

Reply:
done [03d0aac](https://github.com/trezor/trezor-suite/pull/10033/commits/03d0aac57c0a1c4d98a6de68448826379ef5195f)